src/library/scala/collection/StringOps.scala

```diff
@@ -628,9 +628,12 @@ final class StringOps(private val s: String) extends AnyVal {
     *  end characters; i.e., apply `.stripLineEnd` to all lines
     *  returned by `linesWithSeparators`.
     */
-  def lines: Iterator[String] =
+  def linesIterator: Iterator[String] =
     linesWithSeparators map (_.stripLineEnd)
 
+  @deprecated("Use linesIterator. JDK 11 introduces its own lines method on String", "2.13.0-M5")
```

Comment:
`2.13.0-RC1` or `2.13.0` ?

Reply:
Yes, should be RC1 now :-) I'll move these commits to their own PR.

Reply:
we normally just put "2.13.0" so we don't have to keep changing it